scripts/build.sh

```diff
@@ -2,4 +2,4 @@
 export REACT_APP_GIT_SHA=$(echo $(git rev-parse  HEAD) | cut -c1-9)
 export REACT_APP_VERSION=$npm_package_version
 export SKIP_PREFLIGHT_CHECK=true
-yarn react-app-rewired build
+yarn react-app-rewired --max_old_space_size=4096 build
```

Comment:
?

Reply:
I was getting JS heap memory errors during its CI build, this was a solution to the problem.